detect_secrets/main.py

```diff
@@ -39,10 +40,12 @@ def main(argv=None):
             _scan_string(line, plugins)
 
         else:
-            baseline_dict = _perform_scan(
-                args,
-                plugins,
-            )
+            if args.import_filename:
+                plugins = initialize.merge_plugin_from_baseline(
+                    _get_plugin_from_baseline(args.import_filename), args,
+                )
```

Comment:
Let's move this block inside `_perform_scan`, right after we get the old_baseline.

e.g.

```
def _perform_scan(args, plugins):
    old_baseline = _get_existing_baseline(args.import_filename)
    if old_baseline:
        plugins = initialize.merge_plugin_from_baseline(...)
```

This way, we don't need to incur two file reads. Also, I'm not entirely sure that reading from `stdin` would work twice.